lib/components/narrative/default/default-itinerary.js

```diff
@@ -263,68 +287,81 @@ class DefaultItinerary extends NarrativeItinerary {
         onMouseLeave={this._onMouseLeave}
         role="presentation"
       >
-        <button
+        <ItinerarySummaryWrapper
```

Comment:
We could add an `aria-label` here?

Reply:
I definitely see where you're coming from here, however I think we may be okay without it? This element does have an unmarked interaction, but it also has an accompanying button for screen readers so they can access the same interaction in a more straightforward way! 
<img width="270" alt="image" src="https://user-images.githubusercontent.com/115499534/213332972-f01b01c9-57fc-4bd3-9b04-6130bc7fee9f.png">